From: hayeswang <hayeswang@realtek.com>
Date: Mon, 21 Mar 2011 01:50:30 +0000
Subject: [PATCH 52/71] net/r8169: add a new chip for RTL8105

commit 36a0e6c2d6f3eb59b7a5ddfda63d252a42dba189 upstream.

Add a new chip for RTL8105 whose settings are the same with RTL_GIGA_MAC_VER_30.

Signed-off-by: Hayes Wang <hayeswang@realtek.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/r8169.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/net/r8169.c b/drivers/net/r8169.c
index 156afee..29080e6 100644
--- a/drivers/net/r8169.c
+++ b/drivers/net/r8169.c
@@ -1582,6 +1582,7 @@ static void rtl8169_get_mac_version(struct rtl8169_private *tp,
 		{ 0x7c800000, 0x30000000,	RTL_GIGA_MAC_VER_11 },
 
 		/* 8101 family. */
+		{ 0x7cf00000, 0x40b00000,	RTL_GIGA_MAC_VER_30 },
 		{ 0x7cf00000, 0x40a00000,	RTL_GIGA_MAC_VER_30 },
 		{ 0x7cf00000, 0x40900000,	RTL_GIGA_MAC_VER_29 },
 		{ 0x7c800000, 0x40800000,	RTL_GIGA_MAC_VER_30 },
-- 
1.7.5.4

